renderers/chords_renderer.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const parseVerse = require('../verse.js')['parseVerse'];
+
+class ChordsRenderer {
+  constructor(chordChartJson) {
+    this.chordChartJson = chordChartJson;
```

Comment:
Rename chordChartJson -> verse

Reply:
👍 I had a hard time with names, wasn't specifically sure a chart vs. verse vs. a phrase vs. .... Thanks for the suggestions!